Makefile

```diff
@@ -41,6 +40,7 @@ $(eval pkgtarget = $(TARGETDIR)/$(pkgfullname).apk)
 packages/$(pkgname): $(pkgtarget)
 $(pkgtarget): ${KEY}
 	mkdir -p ./$(sourcedir)/
+	$(eval SOURCE_DATE_EPOCH ?= $(shell git log -1 --pretty=%ct --follow $(pkgname).yaml))
```

Comment:
@kaniini will this work if this target it invoked for multiple packages?  I am unfamiliar with the scoping rules of Make.